ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -192,6 +193,11 @@ export default class ConfirmPageContainer extends Component {
             ethGasPriceWarning={ethGasPriceWarning}
           />
         )}
+        {contentComponent && disabled && (errorKey || errorMessage) && (
```

Comment:
Could we create a variable that contains all of these conditions, like `shouldDisplayWarning`, and use that instead?  Inlining this many conditions doesn't seem as clean.

Reply:
@darkwing shouldDisplayWarning variable is created and contains all of these conditions. 
You can see changes here: https://github.com/MetaMask/metamask-extension/pull/12496/commits/2ed4c9e64143f7064cb95118fc54d2b99a471af2
Is it okey now?